\begin{savequote}[75mm]
Imagine living in a world where a Richter 9 earthquake raged continuously ... At the scale of proteins, Brownian motion is even more furious.
\qauthor{George Oster and Hongyun Wang \citep{Oster2003}}
\end{savequote}

\chapter{Thermal motion of ssDNA in the MspA nanopore}
\label{dna_thermal_motion_mspa}

Several experiments have been carried out in order to quantify the thermal motion of analyte molecules in a nanopore.  ssDNA's passage through a nanopore can be modeled as a drift-diffusion process, where the electrophoretic drift is caused by the applied voltage bias, and diffusion is caused by thermal motion.  The drift can be quantified in terms of an effective charge, while the diffusion can be quantified in terms of an effective diffusion constant.  Therefore, measurement of the effective charge and diffusion constant of ssDNA in MspA provides a complete description of the ssDNA motion.  Due to the extreme speed with which ssDNA translocates through an MspA nanopore, it is difficult to quantify thermal motion using measurements of free translocation.

\section{The capture and escape experiment}

The experimental technique used here involves the use of a large protein tethered to the ssDNA that prevents its translocation through the nanopore.  The work in this chapter is the subject of two papers \citep{Lu2015,Fleming2017}, and the experimental technique is based on preceding work with DNA in $\alpha$-hemolysin nanopores \citep{Wiggin2008}, where the protein Avidin was used as a molecular stop to arrest ssDNA translocation.  Other methods used to measure the effective charge of DNA in $\alpha$-hemolysin nanopores have included unzipping dsDNA hairpins \citep{Sauer-Budge2003,Mathe2004,Lakatos2005} and monitoring the escape of DNA hairpins \citep{Wanunu2008,Lathrop2010}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/NA-ssDNA_fluctuation_in_MspA.pdf}
\caption[The idea of capture and escape]{The left panel shows a NeutrAvidin/ssDNA complex trapped in MspA.  The membrane is in gray, and a voltage bias across the membrane results in an electrophoretic force pulling the negatively-charged ssDNA into the MspA pore.  Thermal motion causes non-equilibrium position fluctuations.  On the right, the potential energy profile is shown schematically.  Thermal fluctuations of a distance greater than $L$ leads to escape of the complex.}
\label{fig:capture_escape_idea}
\end{centering}
\end{figure}

The idea is depicted in Figures \ref{fig:capture_escape_idea} and \ref{fig:capture_escape_event}.  Here, the protein NeutrAvidin is used as a molecular stop that binds to the 3' end of ssDNA via a 3'-biotin (see Appendix \ref{idt_dna} for the chemical structure).  The NeutrAvidin protein is too large to enter the vestibule of MspA, and so the complex cannot translocate.  However, an electrophoretic force proportional to the effective charge and applied voltage bias pulls the ssDNA through the nanopore, effectively trapping the complex in the pore, as shown in the potential energy diagram on the right.  Thermal motion leads to position fluctuations, depicted in Figure \ref{fig:capture_escape_idea} as movement of the ssDNA within the pore.  Eventually, if the bias voltage is appropriately low, thermal motion will lead to escape of the complex from the nanopore.  Using $x$ to denote the position of the ssDNA within the nanopore's constriction (see Figure \ref{fig:capture_escape_idea}), escape happens for $x \geq L$, $L$ being the length of ssDNA that sticks through the nanopore at equilibrium.

\subsection{Theory of drift-diffusion and first-passage}

The stochastic dynamics of the NeutrAvidin-ssDNA complex in MspA can be modeled as a one dimensional drift-diffusion process that obeys the Smoluchowski equation.  The probability that MspA's narrowest constriction is at location $x$ along the ssDNA at time $t$, having started at position $x_0$, written $p(x,t|x_0)$, obeys

\begin{equation}
\pard{p(x,t|x_0)}{t} = \frac{\partial}{\partial x} \left[ \frac{F(x)}{\gamma} p(x,t|x_0) \right] + D \pardd{p(x,t|x_0)}{x}
\label{eqn:drift_diffusion}
\end{equation}

\noindent
where $F(x)$ is the electrophoretic force on the ssDNA as a function of position, and $\gamma$ is the viscous drag.  $\gamma$ can be rewritten in terms of the diffusion constant using the Einstein relation: $\gamma = \frac{k_B T}{D}$.  Making this substitution,

\begin{equation}
\pard{p(x,t|x_0)}{t} = \frac{D}{k_B T} \frac{\partial}{\partial x} \left[ F(x) p(x,t|x_0) \right] + D \pardd{p(x,t|x_0)}{x}
\label{eqn:drift_diffusion2}
\end{equation}

The force $F(x)$ depends on the applied bias voltage and the effective charge of the ssDNA.  In the simplest case, neglecting entropic forces and considering only the electrophoretic force,

\begin{equation}
F(x) = \sigma V
\label{eqn:simple_force}
\end{equation}

\noindent
where $V$ is the bias voltage and $\sigma$ is the effective charge per unit length.  In this case,

\begin{equation}
\frac{1}{D} \pard{p(x,t|x_0)}{t} = \frac{\sigma V}{k_B T} \pard{p(x,t|x_0)}{x} + \pardd{p(x,t|x_0)}{x}
\label{eqn:drift_diffusion3}
\end{equation}

The only free parameters in Equation \ref{eqn:drift_diffusion3} are the effective charge per unit length, $\sigma$, and the diffusion constant, $D$.  Even in the case where entropic effects are considered in the force term, these can be estimated without introducing additional free parameters.  For a discussion, see Appendix \ref{first_passage}.  Using a reflecting boundary condition at $x=0$ and an absorbing boundary condition at $x=L$, signifying escape of the ssDNA from MspA, Equation \ref{eqn:drift_diffusion2} can be solved numerically.  In the special case where the force is adequately described by Equation \ref{eqn:simple_force}, then Equation \ref{eqn:drift_diffusion3} can be solved analytically, as in Appendix \ref{first_passage}.

The constriction of MspA initially starts on the domain $x \in (0,L)$ along the ssDNA, but the absorbing boundary means that, as time $t$ increases, the probability that the ssDNA is still in the nanopore, $\int_0^L p(x,t|x_0) \, dx < 1$.  The probability density for escape of ssDNA from MspA at a given time is given by the probability flux from the boundary (at $x=L$):

\begin{equation}
f(t,x_0) = -\frac{d}{dt} \int_0^L p(x,t|x_0) \, dx
\label{eqn:escape_time_dist}
\end{equation}

Finally, the average escape time, $\langle \tau \, \rangle$, can be calculated as

\begin{equation}
\langle \tau \, \rangle = \int_0^L \left( w(x_0) \, \int_0^{\infty} t \, f(t|x_0) \, dt \right) \, dx_0
\label{eqn:mean_escape_time}
\end{equation}

\noindent
where $w(x_0)$ is the probability distribution function for the initial position.  The analytical solution for the simple case where $F(x) = \sigma V$ is discussed in Appendix \ref{first_passage}.

The mean escape time, along with the escape time distribution, can be directly compared to experiment, and can be used to obtain an estimate for the values of $D$ and $\sigma$.

\section{Experimental methods}

% FIGURE that's like 2.6 but with block diagram of the amplifier, digitizer, computer, and NI-DAQ.  Show temp control.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.9\textwidth]{figures/experiment_setup_capture_escape.pdf}
\caption[Experimental setup for capture and escape]{Schematic diagram of the experimental setup for the capture and escape experiment.  Active voltage control is achieved by running a computer program which reads voltage and current data from a National Instruments DAQ and sets the command voltage by supplying an analog voltage output signal from the DAQ that goes to the external command input of the Axopatch 200B.  The Axopatch directly controls all voltages in the experiment.  Meanwhile, current and voltage data are recorded separately on the computer via the DigiData 1440A.}
\label{fig:setup_capture}
\end{centering}
\end{figure}

\subsection{Setup}

The experimental setup is similar to what was described in Section \ref{experimental_setup}.  However, the capture and escape experiment requires active voltage control.  The idea is to capture many individual molecules, and measure the time it takes them to escape from the nanopore at various low bias voltages.  The setup which makes active voltage control possible is shown in Figure \ref{fig:setup_capture}.

\subsection{Sample preparation}

ssDNA was purchased from Integrated DNA Technologies, Inc. [IDT] with HPLC purification.  Sequences used in the work covered in this chapter are all 27 bases long, and include (dA)$_{27}$ [poly(dA)], (dC)$_{27}$ [poly(dC)], (dT)$_{27}$ [poly(dT)], and (dGdA)$_{13}$dG [poly(dGdA)], each with a 3' biotin (chemical details of biotin attachment in Appendix \ref{idt_dna}).  Sequences are listed in Appendix \ref{sequences}.

NeutrAvidin-ssDNA complexes are created by the slow, dropwise addition of a low concentration of biotinylated ssDNA (\SI{1}{\micro\Molar} in TE, pH 7.6) to a high concentration of NeutrAvidin (\SI{10}{\micro\Molar} in 1x PBS), with rapid mixing.  Since each NeutrAvidin has four binding sites for biotin, this method ensures that most NeutrAvidins have either zero or one ssDNA molecule bound.  Free ssDNA is removed using a \SI{20}{\kilo\dalton} molecular weight cutoff size exclusion spin column.  This step also serves to buffer exchange the complexes into the buffer used in the experiment, which is \SI{1}{\Molar} KCl, \SI{10}{\m\Molar} HEPES, pH \num{8.00}.

\SI{10}{\pico\mole} of NeutrAvidin-ssDNA complexes are loaded into the cis side of the experiment, and the cis side is mixed well.  Active voltage control and current monitoring are implemented using a custom Python program to control the NI-DAQ.  The experiment is enclosed by a copper lid, and maintained at \SI{25}{\celsius}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.8\textwidth]{figures/NA-ssDNA_capture_and_escape_event.pdf}
\caption[Capture and escape experiment schematically]{A typical capture and escape event showing measured current and applied voltage. The cartoon depicts snapshots of the complex in the pore that correspond to the data below.  The complex is captured at \SI{160}{\mV} and held for nearly \SI{200}{\ms}.  The applied voltage is then reduced to \SI{65}{\mV}.  After approximately \SI{670}{\ms}, the complex escapes, as indicated by the current's increase to its open pore value at \SI{65}{\mV}.  The period of time between voltage reduction and escape of the complex is called the ``escape time."  After escape, the voltage is increased to \SI{160}{\mV} to await the next molecule.}
\label{fig:capture_escape_event}
\end{centering}
\end{figure}

\section{Results}

\subsection{Escape time measurements}

In order to measure many individual molecules and gather statistics about their escape times, it is necessary to use a relatively high voltage to ensure a reasonable capture rate.  Here, \SI{160}{\mV} is used for this purpose.  Upon capturing a molecule, the voltage is decreased to a constant, low voltage until the molecule escapes.  This process is shown in Figure \ref{fig:capture_escape_event}, along with data recorded for one molecule of poly(dT).

\begin{figure}[!h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/polyA_escape_histograms.pdf}
\caption[Measured escape times of poly(dA) ssDNA from MspA]{The histogram of poly(dA) escape data at \SI{60}{\mV}, \SI{25}{\celsius}, plotted with linear and logarithmic x- and y-axis.  Data points are shown in black.  Error bars in each bin are $\sqrt{N}$.  An exponential fit with the same time constant of \SI{57.8}{\ms} is shown as the green line in each plot.  The exponential fits well to the shortest $\sim 90\%$ of events, denoted by the green shaded region.}
\label{fig:polyA_escape_times}
\end{centering}
\end{figure}

In order to measure short escape times, it is necessary to use the ``pipette capacitance compensation" on the Axopatch 200B, and to use a small lipid bilayer area in order that the capacitance can be fully compensated.  Without full capacitance compensation, large capacitive current spikes will obscure short escape events ($<$ \SI{2}{\ms}).

Measuring the escape times for many individual molecules provides a distribution of escape times.  The distribution measured for poly(dA) escaping against \SI{60}{\mV} applied bias is shown in Figure \ref{fig:polyA_escape_times}.  Shown in the figure are three ways of plotting the same data with linear and logarithmic axes.

The fit to a single exponential, which is the prediction for a first-passage process (see Appendix \ref{first_passage}), is shown in green.  An exponential is a good fit to at least 90\% of the events, while the number of events with long escape times slightly exceeds the predictions of a single exponential.  A chi-squared fit to an exponential ($\sim e^{-t/\tau}$) is performed by binning the data as shown and using error bars of $\sqrt{N}$ from Poisson statistics, where $N$ is the number of molecules in a given bin.  The ``average escape time" is defined as the best fit value of $\tau$, and error bars on $\tau$ represent one standard deviation obtained from the chi-squared fit.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/homopolymer_escape_data.pdf}
\caption[Measured escape times of homopolymer ssDNA from MspA]{(a) Histograms of the escape times measured for the given NeutrAvidin-ssDNA complexes with \SI{60}{\mV} bias applied.  Error bars in each bin are $\sqrt{N}$.  At a given clamping voltage, the distribution of escape times is approximately exponential (solid lines) for all four ssDNA molecules, but the time constant differs.  This time constant is called the ``average escape time."  (b) Average escape time plotted as a function of bias voltage.  Error bars reflect parameter estimation uncertainty at one standard deviation.  Fits to a first-passage model are shown as dashed lines.}
\label{fig:escape_times}
\end{centering}
\end{figure}

Homopolymer ssDNA of each of the four nucleotides were chosen as analytes in order to measure the effective charge and diffusion constant of each, and to see if there were noticeable differences.  Figure \ref{fig:escape_times} shows the results of the same measurements described above for each of the oligonucleotides [oligos].  Homopolymer dG was not used in order to prevent G-quadruplex formation, and a repeat of (dGdA) was chosen instead.  As demonstrated in Figure \ref{fig:escape_times}a, the average escape times at \SI{60}{\mV} clamping voltage are different for each oligo.  In particular, poly(dT) shows substantially longer escape times than the others.  This trend holds for all clamping voltages used.

Figure \ref{fig:escape_times}b shows a summary of all the escape time data measured.  Each data point is the average escape time at a given voltage, measured by fitting an exponential to a histogram of escape times measured at that voltage.  Each data point represents between several hundred and over a thousand individual capture and escape events.  The vertical (time) axis is logarithmic, so differences between the different homopolymers 
span more than an order of magnitude at some voltages.  Changes of \SI{40}{\mV} in the bias voltage during escape lead to changes in the average escape time that span four orders of magnitude.

Fits to the first-passage model are obtained by performing a chi-squared fit of the numerical solution of Equation \ref{eqn:drift_diffusion2} to the data, allowing the parameters $\sigma$ and $D$ to vary.  The fitting procedure yields values for $\sigma$ and $D$ for each oligo, and these are tabulated in Table \ref{table:fit_values}.

\begin{table}[h]
\caption{Effective charges, electrophoretic driving forces, and diffusion constants extracted from the data.  Parameters are estimated from fit to first-passage model.  Uncertainties reflect one standard deviation and are obtained from chi-squared fitting.}
\label{table:fit_values}
\sisetup{
table-number-alignment = center,
table-figures-integer = 2
}
\centering
\begin{tabular}{
l
S[separate-uncertainty,table-figures-uncertainty = 1]
S[separate-uncertainty,table-figures-uncertainty = 1]
S[separate-uncertainty,table-figures-uncertainty = 1]
}
\hline
{ssDNA Sequence}
& {Effective charge}
& {Driving force}
& {Diffusion constant} \\
{}
& {$Q_{eff} \,$ ($e^{-}/$base)}
& {$F$, at \SI{100}{\mV} (\si{\pico\newton})}
& {$D$ (\si{\micro\meter^2/\s})} \\ [0.5ex]
\hline \hline
\rule{0pt}{3ex}(dT)$_{27}$ & 0.62(2) & 19.7(4) & 53(10) \\ 
(dC)$_{27}$ & 0.56(1) & 17.9(3) & 83(17) \\
(dA)$_{27}$ & 0.53(2) & 17.1(5) & 56(17) \\
(dGdA)$_{13}$(dG) & 0.51(1) & 16.2(5) & 43(13) \\ [1ex]
\hline
\end{tabular}
\end{table}

\subsection{Current blockage measurements}

The capture and escape experiment yields much more information than escape times.  The current blockage caused by the presence of the homopolymer ssDNA is also recorded.  Figure \ref{fig:homopolymer_blockages} shows a composite recording of current versus time for four individual NeutrAvidin-ssDNA complexes held in M3-MspA at \SI{160}{\mV} bias, measured in separate experiments, along with the open pore current for M3-MspA.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.55\textwidth]{figures/homopolymer_block_traces.pdf}
\caption[Current blockage traces for homopolymer ssDNA]{Recordings of current versus time for NeutrAvidin-ssDNA complexes, where the ssDNA is a 3'-biotinylated 27mer of either poly(dA), poly(dT), poly(dC), or poly(dGdA), as indicated.  Data are compiled from four separate experiments, each filtered at \SI{10}{\kHz}.  The scale bar for the time axis indicates that each segment lasts about \SI{25}{\ms}.  For reference, the open pore M3-MspA current is shown at the upper left.  Buffer is \SI{1}{\Molar} KCl, \SI{10}{\milli\Molar} HEPES, pH \num{8.00}, and temperature is controlled at \SI{25.00}{\celsius}.}
\label{fig:homopolymer_blockages}
\end{centering}
\end{figure}

Perhaps most immediately apparent is that the blockage levels are different depending on the identity of the nucleotides.  The homopolymers used in this work each block different amounts of current in M3-MspA.  Due to our ability to probe single molecules individually, we can create a histogram of these current blockage levels for many molecules.  Figure \ref{fig:homopolymer_blockage_hists} shows histograms of the mean blockage currents measured for all four oligos at two different bias voltages: \SI{60}{\mV} and \SI{160}{\mV}.  Current blockages are reported in terms of mean $I/I_0$, the ratio of the measured current with DNA blocking the pore to the measured current for the open pore.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/homopolymer_block_hists.pdf}
\caption[Current blockage histograms for homopolymer ssDNA]{Histograms of mean blockage currents when different NeutrAvidin-tethered oligonucleotides are captured in the M3-MspA pore.  $I/I_0$ is the ratio of mean measured current, $I$, to open-pore current, $I_0$.  Each element in the histogram is an individual molecule.  (a) At \SI{60}{\mV} bias, the blockages are well separated, while at \SI{160}{\mV} bias (b), the distributions are broader, and show significant overlap.}
\label{fig:homopolymer_blockage_hists}
\end{centering}
\end{figure}

The spread in the measured current blockages greatly increases at higher voltages, leading to significant overlap of poly(dA), poly(dC), and poly(dT).  A careful look reveals that the histograms even become multimodal, with indications of a peak emerging at lower values of $I/I_0$.

Figure \ref{fig:polyC_hists} shows a detailed look at the emergence of this deeper-blockage peak as a function of applied voltage.  While the peak is not so apparent in Figure \ref{fig:homopolymer_blockage_hists} due to the fact that the histograms there are mean blockages, the histograms plotted in Figure \ref{fig:polyC_hists} are histograms of the full current trace, sampled at \SI{100}{\kHz} after hardware filtering at \SI{10}{\kHz}, and median filtering with a 100-point window post-acquisition.  Every data point contributes one point to the histogram.  Histograms are plotted at each voltage horizontally, with the current ($I/I_0$) on the vertical axis.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/polyC_current_voltage.pdf}
\caption[Current histograms for poly(dC) ssDNA versus voltage]{Current blockage histograms for \num{30} molecules of poly(dC) are plotted at \SI{5}{\mV} increments from \SI{45}{\mV} to \SI{170}{\mV}.  The histograms show that the mean $I/I_0$ increases with applied voltage, and that the distribution is Gaussian at voltages below about \SI{100}{\mV}.  At higher bias voltages, peaks appear at lower values of $I/I_0$, and the current distribution ceases to be Gaussian.  Two distributions are singled out, and short segments of $I/I_0$ versus time are plotted in insets.}
\label{fig:polyC_hists}
\end{centering}
\end{figure}

At voltages below about \SI{100}{\mV}, the current is Gaussian distributed around a single value.  However, as the voltage increases, the distribution becomes asymmetrical, with a distinct peak or peaks emerging at lower values of $I/I_0$.  Two voltages are singled out in the insets: \SI{90}{\mV} and \SI{170}{\mV}.  The currents plotted in the insets correspond to the y-axis values.  Short traces of current versus time are plotted in order to show the dynamic nature of these current fluctuations.  The \SI{170}{\mV} data seems consistent with the picture of transient deviations from a resting state to deeper-blockage states.  These transient deeper-blockage states last on the order of \SI{1}{\ms} or less, and they get farther away from the main peak as voltage increases.  Poly(dC) is the oligo shown in Figure \ref{fig:polyC_hists}, but all four of the oligos measured in this work show the emergence of a similar deeper-blockage peak.

We speculate that the emergence of this deeper-blockage peak at high biases could be caused by transient interactions between the NeutrAvidin protein and MspA.  Such interactions have been speculated about before by others \citep{Manrao2011} in different contexts.  We propose that at high bias voltages, above \SI{100}{\mV}, the electrophoretic force is strong enough to pull NeutrAvidin into a tightly clamped state where it interacts with MspA and blocks more current.  At low bias voltages, the NeutrAvidin tents to be farther away from MspA more often due to reduced tension in the ssDNA.  The likelihood of NeutrAvidin being in the tightly-clamped state is low at low bias voltages.

An alternative explanation for the emergence of these discrete peaks at high bias could be that the ssDNA interacts in a specific manner with the constriction of MspA, and that these specific interactions are stabilized by the ssDNA being under high tension.  Recent molecular dynamics simulations with ssDNA and MspA have indicated that there are specific interactions between the phosphate backbone of the ssDNA and the pore's constriction that can stabilize high-tension states of the ssDNA \citep{Bhattacharya2016a}.  However, those simulations show that the lifetimes of these states are on the order of \SI{1}{\micro\s} or less, while the fluctuations observed in experiments here are on the order of milliseconds (\SI{170}{\mV} inset of Figure \ref{fig:polyC_hists}).

A final observation about the data in Figure \ref{fig:polyC_hists} is that even though the current at each voltage is normalized to the open pore current at that voltage (and open-pore current is itself non-linear with voltage, see Figure \ref{fig:iv_mspa}a), the fractional blockage $I/I_0$ increases significantly as the bias voltage is increased.  This holds true for all four oligos studied, and it indicates that the ionic conductivity of the blocked pore increases as voltage is increased.  We attribute this to a reduction in the number of nucleotides between the NeutrAvidin and the constriction of MspA as voltage increases, due to a larger force on the ssDNA and increased tension.  Stretching of ssDNA leading to elongation was reported by Stoddart \textit{et al.} in $\alpha$-hemolysin \citep{Stoddart2015}.

\subsection{Current noise measurements}

It is also instructive to examine the noise power spectra of current blockages recorded when ssDNA is trapped in the MspA nanopore.  In light of the current fluctuations to a deeper-blocked state at high voltages, Figure \ref{fig:polyC_fluctuations} shows the current noise power spectral density, $S(f)$, for two individual molecules of poly(dC): one held in the pore at \SI{60}{\mV} bias, and the other at \SI{160}{\mV}, corresponding to the voltages shown in Figure \ref{fig:homopolymer_blockage_hists}.  The power spectrum of the molecule at \SI{60}{\mV} is largely ``white," meaning independent of frequency, from \SI{10}{\Hz} to the filer cutoff near \SI{10}{\kHz}.  The molecule trapped at \SI{160}{\mV} shows an entirely different power spectrum.  The large increase in noise at low frequencies, with a shoulder near \SI{200}{\Hz}, is consistent with a telegraph process where the current fluctuates between two (or more) states \citep{Dutta1981} with a characteristic frequency of about \SI{200}{\Hz} (see Appendix \ref{noise}).  For reference, the open pore noise is plotted in Figure \ref{fig:mspa_noise_intro}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.6\textwidth]{figures/polyC_current_noise.pdf}
\caption[Current fluctuations for single molecule poly(dC) ssDNA]{Current noise power spectral density, $S$, plotted as a function of frequency, for two individual molecules of NeutrAvidin-poly(dC) ssDNA.  One molecule is held at \SI{160}{\mV} bias, and the other at \SI{60}{\mV} bias.  The current fluctuations at high bias show an increase in noise power spectral density similar to telegraph noise.}
\label{fig:polyC_fluctuations}
\end{centering}
\end{figure}

The power spectrum obtained at \SI{160}{\mV} contains valuable information about the process involved in fluctuations to the lower-current state, which we speculate has to do with the NeutrAvidin protein, but the extra noise obscures the baseline level of noise associated with the ssDNA itself.  That baseline level of noise is revealed by the measurement at \SI{60}{\mV}.  The noise power spectra at \SI{60}{\mV} can be compared for all four homopolymers, and they are plotted in Figure \ref{fig:homopolymer_fluctuations}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/homopolymer_noise.pdf}
\caption[Current fluctuations for all homopolymer ssDNA]{Current noise power spectral density for the four oligonucleotides tethered to NeutrAvidin and trapped in M3-MspA for at least \SI{5}{\ms} by a \SI{60}{\mV} bias.  The noise power spectral density is averaged over many individual molecules ($n_A=322, n_C=716, n_{GA}=108, n_T=1197$).  The mean white noise, $\bar{S}$, is calculated by averaging the noise from \SI{50}{\Hz} to \SI{500}{\Hz}, labeled the white noise domain.  Capacitive noise begins to dominate above \SI{1}{\kHz}, and produces the visible bump before the filter at \SI{10}{\kHz}.}
\label{fig:homopolymer_fluctuations}
\end{centering}
\end{figure}

Evidently, the noise associated with the ssDNA in MspA is different for each of the oligos in this study.  The noise plotted in Figure \ref{fig:homopolymer_fluctuations} agrees with a qualitative observation of the noise visible on the traces shown in Figure \ref{fig:homopolymer_blockages}, where poly(dGdA) shows visibly more noise than the others, while poly(dT) is the quietest.  The frequency domain of $f \in [50, 500]$\si{\Hz} is appropriate for quantifying the white noise level associated with each oligo, since the molecules are only trapped on the order of \SI{50}{\ms} at an applied bias of \SI{60}{\mV}, and capacitive noise sources begin to dominate at frequencies above \SI{1}{\kHz}.  The mean white noise level between \SI{50}{\Hz} and \SI{500}{\Hz} will be called $\bar{S}$ in the discussion that follows.

\section{Discussion}

\subsection{Thermal motion of the ssDNA in MspA}

Equation \ref{eqn:drift_diffusion2} can be integrated over time to give the time-independent position probability distribution, $P_0(x|x_0) = \int_0^{\infty} p(x,t|x_0) \, dt$.  The solution, with the appropriate reflecting boundary condition at $x=0$ and absorbing boundary condition at $x=L$ is (see Appendix \ref{first_passage}, Equation \ref{eqn:biased_diffusion_solution}).

\begin{equation}
P_0(x) = \frac{k_B T}{\sigma V D} \left[ 1 - \exp{ \left( \frac{\sigma V (x-L)}{k_B T} \right) } \right]
\label{eqn:biased_diffusion_solution2}
\end{equation}

Plots of Equation \ref{eqn:biased_diffusion_solution2} are shown in Figure \ref{fig:position_fluctuations}b for several applied voltage biases.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/calc_position_fluctuations.pdf}
\caption[Position fluctuations of ssDNA in MspA]{(a) Drift and diffusion lead to a process whereby non-equilibrium position fluctuations of ssDNA in a nanopore relax back to equilibrium.  At a bias voltage of \SI{140}{\mV}, a position fluctuation which puts base number 4 in the pore's constriction (red dashed line) will relax back to nearly the equilibrium distribution (solid blue line) in under \SI{10}{\ns}.  The diffusion constant is \SI{5.9e-11}{\m^2/s}, and the force is \SI{17}{\pico\N} at \SI{140}{\mV}, corresponding to an effective charge density of \num{0.54}$e/\text{base}$ on the ssDNA.  These values come from measurements of poly(dA).  (b) At equilibrium, the distribution of ssDNA position depends on the applied force, or equivalently, the applied bias voltage.  Higher voltages lead to tighter clamping of the ssDNA and less thermal motion.}
\label{fig:position_fluctuations}
\end{centering}
\end{figure}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/thermal_motion_in_sequence_data.pdf}
\caption[Thermal motion averaging and sequencing data]{Thermal motion of ssDNA is consistent with nanopore sequencing data.  The cartoons show how a few example sequences sit in the constriction of MspA.  The plots show measured current as a function of the position of a single base substitution in a homopolymeric strand.  The single base substitution results in a different current than the homopolymer, but only in some locations.  The fact that the width of this peak is never less than two or three bases is consistent with the thermal motion we report.  Sequence-specific currents come from measurements on M2-MspA made available online by Laszlo \textit{et al.} \citep{Laszlo2014}.}
\label{fig:thermal_motion_sequencing}
\end{centering}
\end{figure}

\subsection{Correlations}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.6\textwidth]{figures/charge_blockage_correlation.pdf}
\caption[Correlation between effective charge and current]{Correlation between effective charge and mean blockage current.  The larger the blockage current, $I/I_0$, the smaller the effective charge.  Mean values of $I/I_0$ are measured at \SI{60}{\mV}, as shown in Figure \ref{fig:homopolymer_blockage_hists}, and the associated small error bars are the standard error of the mean.  Error bars on effective charge are one standard deviation uncertainty from the first-passage model parameter estimation.}
\label{fig:charge_current_correlation}
\end{centering}
\end{figure}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/diffusion_noise_correlation.pdf}
\caption[Correlation between diffusion and noise]{Correlation between diffusion constant, $D$, and the mean normalized excess white noise, $\bar{N}$, defined in the text.  (a) The four oligonucleotides in this study plausibly fall along the gray curve, denoting $D \propto 1/\bar{N}$.  Large error bars in $D$ reflect parameter estimation uncertainty at one standard deviation.  (b) Further exploration of this correlation, using measurements of poly(dT) at various temperatures.  The gray curve is the same curve plotted in (a).}
\label{fig:diffusion_noise_correlation}
\end{centering}
\end{figure}

\subsection{Implications}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.8\textwidth]{figures/cell.pdf}
\caption[Thermal motion averaging with voltage]{Model of position fluctuations of ssDNA in MspA, and its effects on signal averaging, as a function of applied bias voltage.  Traces of TTTTT to XXXXX at various voltages...?}
\label{fig:thermal_motion_projections}
\end{centering}
\end{figure}
