\begin{savequote}[75mm]
When asked if his painting was inspired by nature, Jackson Pollock replied, ``I am nature."
\end{savequote}

\chapter{The nanopore as a DNA sequencing technology}
\label{dna_sequencing}

Some data obtained from nanopore DNA sequencing experiments has already been presented to provide evidence that thermal motion of ssDNA contributes to the ionic current signal.  This chapter briefly reviews recent developments which enabled nanopores to be used as a single-molecule DNA sequencing technology, in order to provide a background for the experiments covered in the rest of this work.

\section{Inspired by nature's ion channels}

The initial idea to use nanopores as single-molecule sensors for sequencing nucleic acids was independently proposed by Dave Deamer and George Church around 1989 \citep{Deamer2016,Branton2008}.  The inspiration was that the opening and closing of ion channels in biology is often dependent on small peptide domains blocking the channel.  If small peptide domains can block the channel, so could nucleic acids, and the different sizes of the different nucleic acids could potentially cause different current blockages, enabling a direct readout of the sequence \citep{Branton2008}.  This general idea was mentioned in the Introduction and depicted in Figure \ref{fig:nanopore_seq_expectation}.  Since the initial idea, the last three decades have seen an explosion of research activity into the use of nanopores for DNA sequencing.  Here, a few of the major milestones are discussed.  For a more comprehensive historical review, refer to Reference \citenum{Deamer2016}.

\subsection{Developments with $\alpha$-hemolysin from 1990 - 2009}

Early work studying DNA in nanopores was carried out using the protein nanopore $\alpha$-hemolysin, a transmembrane protein with a $\sim$ \SI{5}{\nm} long beta barrel of approximately \SI{1.5}{\nm} diameter that extends through the membrane (see Figure \ref{fig:mspa_ahl}a).  The first demonstration that an electric field can drive ssDNA and ssRNA through $\alpha$-hemolysin was performed by Kasianowicz \textit{et al.} and published in 1996 \citep{Kasianowicz1996}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.65\textwidth]{figures/mspa_ahl_comparison.pdf}
\caption[Comparison between geometries of MspA and $\alpha$-hemolysin]{Vertical cross-sections (yellow) showing the interior of the solvent-accessible surfaces (green) of (a) $\alpha$-hemolysin [Protein Data Bank: 7AHL] and (b) MspA [Protein Data Bank: 1UUN].  The extended beta-barrel of $\alpha$-hemolysin is about \SI{5}{nm} long, which results in ionic current being averaged over all the nucleotides which reside there.  In contrast, MspA's much shorter constriction leads to less signal averaging.}
\label{fig:mspa_ahl}
\end{centering}
\end{figure}

Discrimination between pyrimidine and purine segments in ssRNA on the basis of ionic current was demonstrated in 1999 by Akeson \textit{et al.} \citep{Akeson1999}.  Meller \textit{et al.} followed up with two works in 2000 and 2002 that showed that $\alpha$-hemolysin can discriminate between DNA poly(dA)$_{100}$ and poly(dC)$_{100}$ on the basis of ionic current \citep{Meller2000, Meller2002}.

Around that same time, work was also carried out using hairpin DNA in order to prolong the ionic current signals as compared to free translocation (\textit{translocation} being the movement of a molecule through the nanopore from the cis to trans side of the membrane).  The narrow passage through $\alpha$-hemolysin cannot accommodate dsDNA, so a DNA hairpin can be trapped and held in the nanopore, unlike ssDNA which translocates quickly.  In 2001, Vercoutere \textit{et al.} demonstrated the ability of $\alpha$-hemolysin to discriminate between DNA hairpins of different lengths on the basis of ionic current and blockage duration \citep{Vercoutere2001}.  Discrimination between the identity of individual base pairs at the end of blunt DNA hairpins in $\alpha$-hemolysin followed in 2003 \citep{Vercoutere2003}.

DNA hairpins trapped in a nanopore can be examined at length, when a free ssDNA overhang is captured in the pore.  In 2005, work by Ashkenasy \textit{et al.} demonstrated differences in ionic current through $\alpha$-hemolysin when individual nucleotide \texttt{C} to \texttt{A} substitutions were made in the region of the ssDNA hairpin tail that was dangling in the narrow beta-barrel of the pore \citep{Ashkenasy2005}.

These studies showed that one promising avenue for obtaining sequence-dependent ionic currents was to trap and hold a strand of ssDNA in the nanopore.  Aside from DNA hairpins, DNA-binding proteins could be used to perform this function.  A protein, which would be too large to translocate through the nanopore, would hold the ssDNA in place.  A step in this direction was taken by Astier \textit{et al.} in 2007, where it was shown that the ionic current blockages caused by motor protein P4 - RNA complexes in an $\alpha$-hemolysin nanopore could be distinguished from the RNA alone on the basis of the duration of the current blockage \citep{Astier2007}.  Around the same time, Benner \textit{et al.} demonstrated the detection of the Klenow fragment of DNA polymerase I bound to DNA in $\alpha$-hemolysin, and even showed that the addition of dNTPs would cause the polymerase to function, causing the current blockage events to last even longer \citep{Benner2007}.  Hornblower \textit{et al.} demonstrated the detection of ssDNA - exonuclease I complexes in an $\alpha$-hemolysin nanopore, also in 2007 \citep{Hornblower2007}.  These studies identified the presence of the protein by the fact that the protein - nucleic acid complexes would cause current blockages of longer duration than the nucleic acid alone, but sequence-specific signals or control over the motion of the nucleic acid was yet to be demonstrated.

Then in 2008, a large step was made by Cockroft \textit{et al.} in demonstrating single-nucleotide movements of a DNA polymerase on DNA using abasics as reporters \citep{Cockroft2008}.  The abasic moieties (segments of polyethylene glycol) block far less ionic current than regular nucleotides, and so as the polymerase takes a step, it positions more abasics in the nanopore's beta barrel, and step-wise changes in ionic current can be detected using the $\alpha$-hemolysin nanopore.

Using streptavidin protein bound to a biotinylated ssDNA oligo, Stoddart \textit{et al.} in 2009 showed discrimination between individual nucleotides at certain locations in a DNA strand immobilized in $\alpha$-hemolysin \citep{Stoddart2009}.  Together, these studies demonstrated that ssDNA could be trapped in the nanopore, that it might be able to be moved through a nanopore in a controlled way, and that the ionic current does in fact have the potential to provide sequence-specific information, even at the single-nucleotide level.

\subsection{Foundational work with MspA}

The crystal structure of MspA, obtained in 2004 by Faller \textit{et al.} \citep{Faller2004}, revealed that this nanopore had a shape which seemed very interesting for the purposes of nanopore DNA sequencing.  Shown again in Figure \ref{fig:mspa_ahl}b, the narrowest constriction is much shorter than the constriction of $\alpha$-hemolysin, leading to the conjecture that it might average over fewer nucleotides at a time.  However, unlike $\alpha$-hemolysin, the wild-type MspA nanopore has negative charges lining the constriction and does not translocate ssDNA.  Initial engineering of MspA to translocate ssDNA was carried out in 2008 by Butler \textit{et al.} \citep{Butler2008}.  That study resulted in the creation of M2-MspA, one of the variants of MspA used in this work.

After MspA was engineered so that ssDNA could translocate, a 2010 study by Derrington \textit{et al.} followed up using DNA hairpins to show that homopolymers could be distinguished on the basis of ionic current using MspA \citep{Derrington2010}.  The following year, the NeutrAvidin protein was used to hold ssDNA in place (via a biotinylated ssDNA end), trapping it in the pore.  That work, by Manrao \textit{et al.}, demonstrated that not only are the homopolymer current levels distinguishable, but MspA also provides enough sensitivity to distinguish the positions of single base substitutions in homopolymers if those substitutions are in a specific region of about 4 nucleotides \citep{Manrao2011}.  This showed that MspA is sensitive to approximately 4 nucleotides in its narrowest constriction.


\section{DNA enzymes enable measurement of sequence information}

In an important review on nanopore sequencing written by Branton \textit{et al.} in 2008 \citep{Branton2008}, it was noted that a key challenge to DNA sequencing with a nanopore at the time was finding methods to slow down and control DNA translocation.  DNA enzymes had just begun to be used in nanopore experiments the year before \citep{Astier2007, Hornblower2007}, and showed promise for at least slowing down DNA translocation.  Around the same time the review was written, the study mentioned above by Cockroft \textit{et al.} \citep{Cockroft2008} made a first step toward demonstrating control over the motion of DNA in a nanopore.  This study was extended by Chu \textit{et al.} in 2010 \citep{Chu2010}.

Other DNA polymerases were also studied.  T7 DNA polymerase was shown to replicate DNA molecules in an $\alpha$-hemolysin nanopore \citep{Olasagasti2010}.  This was the first instance of using a strategy to inhibit DNA replication in the bulk phase by using blocking oligos that are peeled off by the pore.  Individual steps of the polymerase could be seen, though not in great detail, using abasic moieties as mentioned above.

The same strategy was employed to great effect using Phi29 DNA polymerase by Lieberman \textit{et al.} in 2010.  Phi29 DNA polymerase was shown to replicate full DNA molecules in $\alpha$-hemolysin, and individual, discrete current levels could be seen, and looked similar from one molecule to the next.  The Phi29 polymerase turned out to be more processive than T7 DNA polymerase \citep{Lieberman2010}, which is an important point, since the experiment ends when the enzyme detaches from the DNA.

Following up on this work, a fascinating study in 2012 by Cherf \textit{et al.} demonstrates that Phi29 DNA polymerase can be operated both as a voltage-driven unzipper and an enzymatically-driven ratchet (as it polymerizes) \citep{Cherf2012}.  Phi29 was shown to take single-nucleotide steps on DNA, which can be reversed by voltage-mediated unzipping.  This work was carried out using $\alpha$-hemolysin, using abasic residues to get a large current signal.

The major step that remained was the combination of the benefits of the MspA nanopore with the DNA enzyme ratcheting using Phi29.  Such a study was conducted by Manrao \textit{et al.} in 2012 \citep{Manrao2012}.  Phi29 DNA polymerase on DNA was demonstrated to produce sequence-specific current levels in MspA that corresponded to single-nucleotide steps.  Importantly, the steps in ionic current were due to normal DNA nucleotides rather than the presence versus absence of abasic moieties.  In 2014, Laszlo \textit{et al.} \citep{Laszlo2014} followed up with a more comprehensive study of Phi29 DNA polymerase on DNA in MspA, and created a library of current levels in MspA for every possible 4-base combination of nucleotides that block MspA's narrow constriction [\texttt{AAAA}, \texttt{AAAC}, \texttt{AACA}, ... \texttt{TTTT}].  This library was used to read a DNA sequence.

By this time, in 2014, Oxford Nanopore Technologies, Inc., had released its MinION nanopore DNA sequencer to a select group of researchers.  The sequencer, approximately one by four by one inches in physical size, contains an array of 2000+ individual nanopores in separate membranes, each electronically controlled and monitored, and connects via USB to a computer.

Further work in the literature has made use of other kinds of DNA enzymes, recently the helicase Hel308.  This helicase can also be used with MspA to obtain sequence information \citep{Derrington2015}.  In the work presented here, a Dda helicase is used to control the motion of DNA through the MspA nanopore.  The use of a Dda helicase to ratchet DNA through a nanopore base-by-base has not yet appeared in the literature.

\subsection{DNA helicases}

The helicase used in this work is Dda (first identified in 1984 by Jongeneel \textit{et al.} \citep{Jongeneel1984}), a monomeric SF1B DNA helicase that takes a single nucleotide step in the 5' to 3' direction \citep{Raney1995} when it hydrolyzes a single molecule of ATP.  As it steps, it unzips the dsDNA duplex ahead of it, and stabilizes the unzipped form of the DNA.  The general action of this helicase, and others like it, is depicted in Figure \ref{fig:helicase_drawing}.  Helicases perform a critical role in cells, allowing for ssDNA to be exposed and transcribed to mRNA as well as to be copied prior to cell division.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.6\textwidth]{figures/helicase_drawing.pdf}
\caption[DNA helicase general function]{Cartoon diagram showing the general function of a DNA helicase, and the way it interacts with DNA.  DNA helicases unwind dsDNA to form separated strands of ssDNA under the action of ATP.  The helicase shown here is monomeric, like Dda helicase, and this helicase grips one of the ssDNA strands and steps along it, a distance of one nucleotide per ATP molecule hydrolyzed in the case of Dda helicase.  Other helicases can be multimeric, and can have different mechanisms of action.}
\label{fig:helicase_drawing}
\end{centering}
\end{figure}

In 2012, He \textit{et al.} \citep{He2012} obtained a crystal structure of Dda helicase bound to a short stretch of ssDNA, poly(dT)$_8$ (Protein Data Bank: 3UPU).  A stereo pair image of that crystal structure is shown in Figure \ref{fig:helicase_stereo}.  The structure is angled so that the ssDNA molecule points out of the page, with the 3' end out.  The different domains are labeled in different colors.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_structure_stereo.png}
\caption[Dda helicase structure]{Stereo image of the crystal structure of Dda helicase bound to ssDNA, in gray.  The 3' end of the ssDNA is sticking out of the page.  Domains of the protein are shown in different colors and labeled.  Protein Data Bank number 3UPU. \citep{He2012}}
\label{fig:helicase_stereo}
\end{centering}
\end{figure}

The different domains of Dda helicase are thought to perform different functional roles.  Domains 1A and 2A primarily interact with the negatively-charged phosphate backbone of the ssDNA.  Domain 2B and the hook (in green) stabilize the position of domain 1B, called the ``pin."  The pin domain, remarkably, acts as the pawl on a ratcheting mechanism.  Figure \ref{fig:helicase_ssDNA_interactions} shows the interactions between the helicase and the ssDNA in more detail.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/helicase_dna_interaction.png}
\caption[Dda helicase: interactions with ssDNA]{A cutaway view of Dda helicase (Protein Data Bank 3UPU), showing the ssDNA in its groove.  The major interactions of the protein with the ssDNA are shown by drawing the side-chains involved using a stick representation.  Most of the interactions are between two particular $\alpha$-helices and the ssDNA phosphate backbone.  The two $\alpha$-helices are in different domains of the protein, and move relative to each other when ATP binds.  The other important interaction is the hydrophobic stacking of two phenylalanine rings from the pin (yellow) with the ssDNA bases.}
\label{fig:helicase_ssDNA_interactions}
\end{centering}
\end{figure}

Most of the interactions of Dda helicase with ssDNA are between polar amino acids and the ssDNA phosphate backbone \citep{He2012}.  These interactions are shown in Figure \ref{fig:helicase_ssDNA_interactions} in orange, cyan, and blue by drawing the amino acids involved using a stick representation.  Some hydrophobic stacking interactions also exist between amino acid side chains and the ribose sugars of the ssDNA \citep{He2012}, which are not shown.

The remarkable part of the helicase is domain 1B, the ``pin."  The two phenylalanine rings in the pin, shown in yellow in stick form in Figure \ref{fig:helicase_ssDNA_interactions}, stack with ssDNA base 6 (counting from base 1 at the 5' end).  This hydrophobic stacking interaction acts as the pawl in a ratchet mechanism, preventing the ssDNA from moving backwards.  Here, backwards refers to a movement of the entire ssDNA strand toward the 3' end, toward the right, where the dsDNA duplex would be located.  Effectively, the pin prevents the unwinding duplex from re-hybridizing.  If the pin is mutated, changing the base-stacking phenylalanine to alanine, which removes this hydrophobic stacking, then the helicase is unable to unwind dsDNA.  However, this mutation does not prevent the helicase from translocating along ssDNA \citep{He2012}.

Though there is no crystal structure of Dda helicase bound to ssDNA and ATP, it is believed that ATP binds in a pocket between domains 1A and 2A \citep{He2012}.  The binding of ATP, and subsequent hydrolysis and release of ADP, cause conformational changes that lead to movement of the helicase in the 5' to 3' direction along the ssDNA by one nucleotide.  In Figure \ref{fig:helicase_ssDNA_interactions}, the conformational change induced by ATP binding corresponds to a movement of helix 2A$\alpha$3 to the right, toward helix 1A$\alpha$3.  Instead of pointing at base 2, as shown, the movement causes helix 2A$\alpha$3 to points at base 3.  Helix 2A$\alpha$3 remains associated with base base 3 as ATP is hydrolyzed and ADP is released, and as the helices move apart again upon ADP release, helix 1A$\alpha$3 associates with base 6 rather than its initial partner, base 5.

The result is that Dda helicase moves along ssDNA in the 5' to 3' direction, in an ``inch-worm"-like mechanism \citep{Spurling2006}, by hydrolyzing ATP, unwinding dsDNA duplex as it goes \citep{Byrd2012}.  Dda can even displace bound proteins, including streptavidin bound to a 3' biotinylated oligonucleotide \citep{Morris1999}, a result which we confirmed experimentally using a gel electrophoresis assay (data not shown). %in Appendix \ref{click_reaction_quad}.
