%\begin{savequote}[75mm]
%blank
%\qauthor{blank}
%\end{savequote}

\chapter{Measuring helicase movement along DNA using a nanopore}
\label{helicase_on_dna}

The review of the recent progress in nanopore research given in Chapter \ref{dna_sequencing} revealed that two key developments have enabled DNA sequencing: (1) the use of a DNA enzyme to control the motion of the DNA, and (2) the use of a nanopore with a physically short and narrow constriction, such as MspA.  This chapter presents data obtained using the MspA nanopore and the DNA helicase Dda to control the motion of ssDNA though the nanopore.

\section{Experimental setup}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.95\textwidth]{figures/helicase_scheme.pdf}
\caption[Helicase can control DNA translocation]{(a) Helicase is bound to ssDNA at the junction where the ssDNA meets an unzipping duplex.  (b) The scheme for using a helicase in a nanopore experiment.  The helicase, perched atop the nanopore, is too large to fit into the pore.  Because it is bound to the ssDNA strand, the helicase prevents the DNA from translocating.}
\label{fig:helicase_scheme}
\end{centering}
\end{figure}

Figure \ref{fig:helicase_scheme} depicts the experimental scheme for using a helicase enzyme to slow the translocation of DNA through the nanopore, from the cis to the trans side of the lipid membrane.  The helicase, bound to the ssDNA strand at the ssDNA – dsDNA junction, unwinds the dsDNA duplex by stepping forward along the ssDNA and stabilizing the open conformation of the helix.  The helicase moves by catalyzing the hydrolysis of ATP.  When the free 5’ end of the ssDNA strand is pulled into the MspA nanopore, the helicase ends up perched on top of MspA.  The helicase is too large to fit into the vestibule of MspA, and so it is clamped in place by the electrophoretic force which pulls the DNA through the nanopore.  The extent of this clamping, and the thermal motion of the helicase in this clamped state, was explored in Chapter \ref{dna_thermal_motion_mspa}, where now the helicase is playing the role previously played by the NeutrAvidin protein.

Of course, unlike the NeutrAvidin protein, the helicase is able to step along the ssDNA.  This helicase is always oriented with its front facing the 3’ end of the ssDNA strand to which it is bound (see the crystal structure in Figures \ref{fig:helicase_stereo} and \ref{fig:helicase_ssDNA_interactions}).  It will be shown that the helicase is capable of moving forward (5’ to 3’) and backward (3’ to 5’) along ssDNA; however, the steps associated with hydrolysis of ATP are thought to be exclusively in the forward (5’ to 3’) direction.

\section{Data from a helicase / DNA experiment}

The experimental scheme depicted in Figure \ref{fig:helicase_scheme} was used to obtain the ionic current data shown in Figure \ref{fig:helicase_data}.  The attachment of helicase to DNA is described in Appendix Section \ref{sample_prep}, and the DNA used is listed in detail in Appendix \ref{sequences}, sequence \# 0001, with its complement and tether.  As shown in Figure \ref{fig:helicase_scheme}, the DNA used is a hairpin, with a section of dsDNA duplex, \num{50} base pairs, a binding site for the Dda helicase, poly(dT)$_{12}$, and an overhang of abasic spacer ssDNA, \num{30} nucleotides in length, where the helicase cannot bind.

\begin{figure}[H]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_data.pdf}
\caption[Data for helicase stepping along DNA]{(a) Current recorded when one particular Dda helicase / ssDNA complex is held in the M2-MspA nanopore at \SI{120}{\mV} bias using the experimental setup in Figure \ref{fig:helicase_scheme}b, with ATP powering the helicase.  (b) Same data, but with the y-axis zoomed in and the data discretized into stable levels (shown in red) with sudden jumps between them.  Buffer is \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, pH \num{8.00}, with \SI{2}{\m\Molar} MgCl$_2$ and \SI{2}{\m\Molar} ATP.}
\label{fig:helicase_data}
\end{centering}
\end{figure}

The buffer used in the experiment is \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, pH \num{8.00}, with \SI{2}{\m\Molar} MgCl$_2$ and \SI{2}{\m\Molar} ATP.  The helicase will not unwind the duplex DNA in bulk solution due to the fact that there is a stretch of \num{4} abasic spacers in front of the helicase, between it and the duplex, which prevent its normal function.  Once a helicase/DNA complex is pulled into the nanopore however, the electrophoretic force pulls the helicase past this abasic region and the helicase is then able to function normally with ATP.  Approximately \SI{1}{\pico\mol} of the helicase/DNA complex is loaded into the cis side buffer solution (~\SI{200}{\micro\liter} total volume) and mixed well.  The DNA preferentially sticks in the lipid membrane due to a cholesterol tether which is attached to the DNA strand via a flexible linker.

Figure \ref{fig:helicase_data} shows an ionic current trace recorded when one such complex is captured in the M2-MspA nanopore at a \SI{120}{\mV} bias (trans side positive with respect to cis).  Raw data obtained with a \SI{10}{\kHz} hardware filter is in gray, while a \SI{1}{\kHz} filtered version of the data is shown in blue.  The open pore current can be seen near the top of the current axis in Figure \ref{fig:helicase_data}a at the beginning and end of this particular event.  The ionic current is seen to drop, change between what appear to be many discrete, stable levels, all lasting different amounts of time, and then finally the the current returns to the open pore level.  This corresponds to capture of the ssDNA free end with the helicase preventing unchecked translocation, followed by different current levels due to different nucleotides in the nanpore's constriction as the helicase unwinds the duplex and feeds ssDNA through the nanopore.

A slightly larger plot of the same data is presented in Figure \ref{fig:helicase_data}b.  The current blocks to a level near \SI{100}{\pA} initially, then falls to near \SI{20}{\pA}, then climbs back up to \SI{100}{\pA} and back down.  The relatively high current levels correspond to abasic moieties in the nanopore's constriction, while the data after about \SI{2.5}{\s} correspond to currents measured with combinations of the nucleotides \texttt{A}, \texttt{T}, \texttt{C}, and \texttt{G} in the nanopore's constriction.  Remarkably, this experiment provides enough signal-to-noise to distinguish different current levels corresponding to different DNA sequences.

\begin{figure}[H]
\begin{centering}
\includegraphics[width=0.8\textwidth]{figures/helicase_stepping_three_molecules.pdf}
\caption[Helicase stepping along three DNA molecules]{Data from three individual molecules.  The current levels shown are those that correspond to the same region of the DNA molecule as the current levels from about \SI{2.3}{\s} to about \SI{4.5}{\s} in the data shown in Figure \ref{fig:helicase_data}.  The discrete levels have been labeled with the numbers \num{1} though \num{17} in each dataset.  The length of time each stable current level lasts, as well as the presence or absence of particular levels, gives a feel for the variability between individual molecules.}
\label{fig:helicase_data_3}
\end{centering}
\end{figure}

Plots of data recorded in the same way for three individual helicase/DNA complexes are shown one on top of another in Figure \ref{fig:helicase_data_3}.  Here ionic current is displayed normalized by the open pore current, $I_0$.  In order for these ionic current levels to correspond to DNA sequence, they should be reproducible molecule-to-molecule for individual helicase/DNA complexes on strands of DNA with the same sequence.

It turns out that the current levels are reproducible from one molecule to the next.  The solid color lines and numeric annotations in Figure \ref{fig:helicase_data_3} show unique ionic current levels that occur in a specific order.  These levels can be compared between the three molecules shown.  These three current traces were selected at random from a larger dataset.  The timing of each current level is very different (note the separate time axes), and while the same levels usually appear in all three molecules, some are skipped over in one particular recording or another.  Some of the levels also differ slightly in the ionic current value from one molecule to the next, but many are nearly exactly the same.

\section{Data analysis}

The nanopore experiment described here provides a large amount of high-quality data which encodes a DNA sequence in ionic current levels.  Extracting information from these measured current levels is an interesting challenge in data analysis.  For example, the discrete current levels shown in Figures \ref{fig:helicase_data} and \ref{fig:helicase_data_3} were obtained using a level-finding algorithm discussed in Appendix \ref{level_finding} that essentially determines where a level transition occurs by attempting to minimize the within-level variance while maximizing the between-level variance.  The identities of the various levels annotated in Figure \ref{fig:helicase_data_3} were determined by hand.  When working on a large dataset, it becomes necessary to automate not only level-finding but also level alignment.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_levels_several_mols.pdf}
\caption[Current levels are reproducible molecule to molecule]{Data from 19 molecules from the same experiment shown above.  Current levels are now represented by their means, and time information is removed.  The levels in all 19 molecules have been aligned to the data from one particular molecule, and individual molecules' levels are shown as colored circles.  The mean consensus levels after alignment are shown in black.  The bar graph at the bottom shows how many of the molecules each level appears in.}
\label{fig:helicase_data_aligned}
\end{centering}
\end{figure}

Data from \num{19} individual helicase/DNA complexes, all with the same DNA sequence, are displayed in Figure \ref{fig:helicase_data_aligned}.  Here both level-finding and level-alignment were performed automatically.  The level-alignment algorithm models the process using a hidden Markov model, where the Markov states correspond to locations of the helicase along the DNA, and the current levels correspond to the observables in each hidden Markov state.  The most probable underlying hidden states, as well as the most probable path through them, can be computed by means of the dynamic programming approach called the Viterbi algorithm.  Further detail is contained in Appendix \ref{level_alignment}.

The current level sequences in Figure \ref{fig:helicase_data_aligned} were all aligned to one individual sequence of current levels, without any prior knowledge.  The data are plotted with the time information removed, and the mean current of a given level plotted against its position in the sequence of levels after alignment (its ``level number").  Data from each molecule is shown in color, while the black circles with error bars denote the mean of these mean current levels for each molecule, as well as the standard deviation.  Shown at the bottom is a bar graph that denotes how many of the \num{19} molecules had a measured current level that aligned to the given level number.  Most of the levels \num{1} through \num{30} appear to be present in almost every molecule.  As the level number increases, the number of molecules that exhibit a given current level decreases.  This is likely due to some molecules ending early, if the last bit of dsDNA duplex melts prematurely.  On ssDNA, the helicase can be suddenly pushed past several nucleotides at once, causing the helicase to fall off the end of the DNA, skipping the last several current levels.

\section{Movement of helicase on DNA is revealed by current levels}

Data have demonstrated that discrete, stable current levels are observed when Dda helicase is bound to DNA and allowed to unwind dsDNA using ATP, feeding the ssDNA into the M2-MspA nanopore as it unwinds.  It is instructive to take a closer look at the setup in order to understand exactly where these different current levels come from.  Figure \ref{fig:helicase_sequencing} shows the experimental scheme again, along with data that might be measured.

In the configuration shown in Figure \ref{fig:helicase_sequencing}a, an ATP-driven step of the helicase will result in the helicase being positioned one base closer to the 3’ end of the DNA, as shown in panel (b), and a single base of the duplex will have been unzipped in the process.  As soon as the helicase moves, the electrophoretic force on the DNA causes the DNA to advance through the MspA nanopore by one base.  Before the ATP-driven step, the 5 base sequence near the narrow constriction of MspA was \texttt{ATTCG}.  The 5-base sequence near the nanopore’s constriction is referred to as the ``5-mer."  After the ATP-driven step, the 5-mer is \texttt{TTCGC}.  As shown in Figure \ref{fig:helicase_sequencing}c, the measured current depends on the 5-mer, and there is an abrupt step in the current when the helicase takes a step.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.95\textwidth]{figures/helicase_sequencing.pdf}
\caption[Sequencing DNA with the help of a helicase]{(a) Initial position of helicase on DNA.  The five ssDNA bases in the nanopore constriction are \texttt{ATTCG}.  (b) After the helicase takes a single, ATP-driven step, the five bases in the nanopore constriction are \texttt{TTCGC}.  (c) Schematic of measured current that would correspond to the situations in panels (a) and (b) shows a sharp jump between two stable mean current values, which occurs precisely when the helicase takes a step.}
\label{fig:helicase_sequencing}
\end{centering}
\end{figure}

Thus the current versus time data provides information about the location of the helicase along the DNA strand at any given time.  Tracking the sequence of current levels is equivalent to tracking the location of the helicase.  After finding the discrete current levels and aligning them to a sequence of known current levels that correspond to the DNA sequence, the current versus time can be mapped to helicase location versus time.  A plot showing helicase location versus time is shown in Figure \ref{fig:helicase_movement}a, where the data for \num{15} individual molecules are overlaid.  It can be seen that the movement of the helicase under the action of ATP is stochastic, and so each molecule follows its own trajectory.  Some molecules pause in one location or another for an extended period of time.  Careful inspection of the plot in Figure \ref{fig:helicase_movement}a reveals that, very occasionally, the helicase even takes a backward step.  

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_position_time.pdf}
\caption[Current levels reveal helicase movement]{(a) Current versus time data can be converted to helicase location versus time by means of current level discretization and alignment to a known sequence of current levels.  Helicase location is plotted here as a function of time for \num{15} individual molecules with the same DNA sequence, from the same experiment.  Conditions were \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, pH \num{8.00}, with \SI{2}{\m\Molar} MgCl$_2$ and \SI{2}{\m\Molar} ATP, at \SI{23}{\celsius}.  (b) Histogram of durations of each current level, which is to say, a histogram of the step times of the helicase.  Apparently, much of the data is distributed as an exponential with a time constant of about \SI{6}{\ms} (red line), but there also exists an excess of levels out in the long-time tail of the distribution.}
\label{fig:helicase_movement}
\end{centering}
\end{figure}

The timing of the helicase's steps is shown in Figure \ref{fig:helicase_movement}b as a histogram, where each aligned current level contributes one element to the histogram.  Most of the levels have a short duration.  The distribution of level durations seems to be approximately exponential, at least at short times, but then there is a long tail to the distribution that is not captured by an exponential.  The exponential fit, shown in Figure \ref{fig:helicase_movement}b as the red line, has a mean of about \SI{6}{\ms}.  However, there are many levels with durations longer than \SI{1}{\s}, which do not follow the exponential distribution.

\section{Discussion}

\subsection{Helicase enzyme kinetics}

From the data, the current levels appear as abrupt (< \SI{10}{\micro\s}) changes between stable levels that can last from less than a millisecond to tens of seconds.  This indicates that the physical movement of the helicase is very rapid, and is not the rate-limiting kinetic step.  Most of the time appears to be spent waiting for one or more steps involved in the enzyme kinetics that do not involve actual movement of the helicase relative to the DNA strand.

It seems possible that the long tail in the distribution of level durations shown in Figure \ref{fig:helicase_movement}b might be explained by invoking multiple states for the enzyme.  Simple transition state theory would predict that a one-step kinetic process catalyzing the hydrolysis of ATP to ADP would lead to an exponential distribution.  It is possible that, instead of having one rate-limiting step, the helicase can be in one of several states, including a normal state, with a single rate-limiting step, and some other sort of ``stalled" state, which involves a much longer wait time.  More data would need to be examined in order to describe the enzyme kinetics quantitatively in terms of a model.

\subsection{Extracting sequence information from current levels}

Because the MspA nanopore averages over approximately five nucleotides, the job of determining which 5-mer gives rise to a measured current level is a difficult task, as there are $4^5 = 1024$ possible 5-mers.  The measured current range spans on the order of \num{30} or \SI{50}{\pA}, depending on the applied voltage, and the current noise can be on the order of one to several picoamperes.  However, the problem again becomes tractable if the \textit{order} of the current levels is taken into account.  As shown in Figure \ref{fig:helicase_sequencing}, the first four bases of any given 5-mer must be the last four bases of the previous 5-mer.  Therefore, from any given state, there are only four possibilities for the following state.  That is to say, the next base must be either \texttt{A}, \texttt{T}, \texttt{C}, or \texttt{G}.  This allows for the most probable DNA sequence to be calculated by observing the entire sequence of current levels, where one level alone is not sufficient to determine the 5-mer which gave rise to it.  Figuring out which sequence of 5-mers was most probable based on a set of measured current levels can be achieved using a hidden Markov model.  For a more thorough treatment of the algorithms used to obtain DNA sequence information, see, for example, Szalay and Golovchenko \citep{Szalay2015}.  Even more recently, Oxford Nanopore and other researchers have begun to use recurrent neural networks as a means for solving this problem of decoding current levels to obtain a most-probable DNA sequence \citep{Boza2017}.

\subsection{Monitoring the movements of a single enzyme}

The experiments described here enable the measurement of the movement of Dda helicase along a DNA strand in single-nucleotide steps.  Since ssDNA is approximately half a nanometer per nucleotide, this experiment provides unprecedented resolution in terms of motion of a single molecule, as was pointed out by Derrington \textit{et al.} \citep{Derrington2015} in their work using the helicase Hel308.  That study claims better spatial and temporal resolution than other techniques including optical tweezers and single-molecule F\"orster resonance energy transfer [FRET].

It is remarkable that the movements of a single helicase enzyme can be tracked with sub-nanometer spatial resolution and sub-millisecond temporal resolution.  This experimental setup makes it possible to contemplate addressing the question: Is it possible to control the motion of a single helicase enzyme along DNA?
