\begin{savequote}[75mm]
All the physical and chemical laws that are known to play an important part in the life of organisms are of [a] statistical kind; any other kind of lawfulness and orderliness that one might think of is being perpetually disturbed and made inoperative by the unceasing heat motion of the atoms.
\qauthor{Erwin Schr\"odinger, \textit{What Is Life?}}
\end{savequote}

\chapter{Controlling the movement of a helicase on DNA}
\label{helicase_motion_control}

\section{Background}

\subsection{Motivation for controlling helicase motion}

One of the main factors that limits the accuracy of a DNA sequence obtained from a single read in a nanopore is the presence of homopolymer regions, or repeated nucleotides, in the DNA.  Repeated nucleotides in stretches longer than the five bases are common in biology \citep{Dechering1998}, but the constriction of MspA reads only about \num{5} nucleotides simultaneously.  This means that as the DNA is unwound by a helicase or other enzyme and fed through the pore base-by-base, the \num{5} nucleotides in the pore remain the same, and the current does not change.  Thus the nanopore misses information about stretches of homopolymers longer than \num{5} nucleotides.

Reads from several identical copies of a DNA molecule can be obtained and the data combined using a variety of algorithms (for a discussion see Reference \citenum{Szalay2015}).  Combining information from \num{100} molecules can yield approximately \num{99}\% accuracy in the DNA sequence.  However, the information about the length of long homopolymer repeats is missing, and this missing information accounts for approximately half of the remaining error \citep{Szalay2015}.  As data quality improve and as the data analysis increasingly takes place using recurrent neural networks \citep{Boza2017}, it is possible that the major source of error that will remain is these long stretches of homopolymers.

If the length of these homopolymer regions of DNA could be determined, it would boost the overall accuracy of DNA sequences obtained with a nanopore.  One strategy is to use the time information obtained from nanopore current recordings.  Since current levels last different amounts of time, it is possible to make inferences about repeats \citep{Sarkozy2017}.  Yet the most desirable solution would be to determine the number of repeats experimentally.  The strategy of the work presented in this chapter is to develop such an approach, based on exerting temporal control over the motion of the helicase enzyme.  If the motion of the enzyme could be precisely timed, and the enzyme made to step at regular intervals, then the number of nucleotides in a homopolymer repeat could be read directly.

\subsection{General considerations about mechanical force in biology}

Much work has been carried out in studying the role of mechanical force in the functioning of motor proteins, ever since the development of single-molecule experimental techniques such as optical tweezers, magnetic tweezers, and atomic force microscopy \citep{Neuman2008}.  The nanopore-DNA experimental setup described here has also been proposed as a sensitive instrument capable of measuring small forces and displacements \citep{Derrington2015}.

Mechanical forces are important in biochemistry, particularly in the functioning of motor proteins.  For an overview, see References \citenum{Bustamante2004} and \citep{Oster2003}.  In general, motor proteins convert chemical energy into mechanical work.  Motor proteins, including helicases, polymerases, and also mysosins (involved in muscle contraction) and kinesins (involved in intracellular cargo trafficking) walk along directional tracks, hydrolyzing nucleotide triphosphates in order to do so.

% Mechanical force is also important in force-sensitive ion channels such as those involved in mechanotransduction, \textit{i.e.} biochemical signaling within a cell in response to mechanical forces \citep{Jaalouk2009}.

An optical tweezer setup was used to measure the force applied by an \textit{E. coli} RNA polymerase during trascription in 1995 (approximately \SI{14}{\pico\N}) \citep{Yin1995}.  Myosin-V was found to act as a mechanical ratchet, rectifying mechanical forces applied to it \citep{Gebhardt2006}.  External forces were applied in the forward direction, assisting the enzyme's motion along actin, as well as in the reverse direction, using an optical tweezer.  The rate of stepping by myosin-V was found to depend on this applied force \citep{Clemen2005}.  Kinesin is capable of producing forces of about \SI{7}{\pico\N} \citep{Higuchi1997}.  Studies on kinesin have demonstrated that force-induced backward steps of kinesin might in fact synthesize ATP, in the reverse reaction of ATP hydrolysis involved in forward stepping \citep{Hyeon2009}.  Synthesis of ATP was demonstrated by Itoh \textit{et al.} by applying a torque to rotate the F$_1$-ATPase in the reverse of its ATP-hydrolysis direction \citep{Itoh2004}.

\subsection{This work}

Mechanical force has been shown by Cherf \textit{et al.} to mediate unzipping of a dsDNA duplex by phi29 DNA polymerase in $\alpha$-hemolysin nanopore experiments \citep{Cherf2012}.  Here, this same idea of force-mediated unzipping of a duplex is explored, using the Dda helicase enzyme.  The strategy is to advance the helicase along DNA using applied mechanical force.  Experiments demonstrate that mechanical force can be used as a substitute for the hydrolysis of ATP in order to force the enzyme to take single-nucleotide steps.  Measurements are made to quantify helicase step times as a function of voltage and temperature.  Attempts are made at temporal control over helicase stepping using brief pulses of applied voltage.


\section{Experiment}

The experimental setup is the same as that shown in Figure \ref{fig:helicase_scheme}b, with M3-MspA in a PC lipid membrane.  A Dda helicase - DNA complex is captured in the nanopore, and a constant voltage bias is applied.  The buffer is typically \SI{1}{\Molar} KCl with \SI{25}{\m\Molar} K-phosphate buffer at pH \num{8.00}.  \SI{2}{\m\Molar} MgCl$_2$ is added to the cis side with the helicase, but no ATP is added.  The only force applied to advance the helicase along the DNA is the electrophoretic force on the DNA induced by the voltage across the membrane.

\section{Mechanical force as a sufficient substitute for ATP}

\begin{figure}[H]
\begin{centering}
\includegraphics[width=0.9\textwidth]{figures/helicase_force_stepping.pdf}
\caption[Mechanical force substitutes for ATP and moves helicase]{(a) Selected segment of a current versus time trace obtained at \SI{37}{\celsius} as a constant \SI{160}{\mV} bias is applied across the membrane.  The Dda helicase is apparently forced to move forward along the DNA strand, unzipping the dsDNA duplex ahead of it.  Note the relatively long time scale on the x-axis.  Gray is the raw data obtained at \SI{10}{\kHz}, while blue is filtered to \SI{1}{\kHz}.  (b) The current level means, with the time information removed, showing the same data.  Current levels measured when the helicase advances due to applied force (red) are aligned to known model levels, measured with ATP (black).}
\label{fig:helicase_force_stepping}
\end{centering}
\end{figure}

Figure \ref{fig:helicase_force_stepping}a shows a short segment of current versus time data for a single DNA molecule, taken as Dda helicase is forced to step along the DNA due to a constant \SI{160}{\mV} applied bias at \SI{37}{\celsius}, unwinding dsDNA duplex ahead of it.  The DNA used in this experiment is sequence \# 0002, along with its complement and tether, listed in Appendix \ref{sequences}.  The same sort of discrete steps in the current are observed as in the case where helicase is driven by ATP (see Figure \ref{fig:helicase_data}, for comparison).  The current level means can be aligned to known model levels for the same sequence obtained while running the helicase forward using ATP, when it steps one base at a time.  The comparison is shown in Figure \ref{fig:helicase_force_stepping}b, and this demonstrates that the helicase is still stepping base-by-base along the DNA.  The DNA sequence used here was intentionally designed to produce a large current variation, which is achieved by transitioning from \texttt{TTTTT}, one of the lowest current levels, to \texttt{GGAAA}, the highest current level.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_levels_sliding.pdf}
\caption[Mechanical force results in the same current levels]{Aligned data from several molecules of identical DNA sequence, showing the current levels measured in M3-MspA (red) as helicase steps along the strand.  The helicase is advanced by the mechanical force applied due to a \SI{160}{\mV} transmembrane potential difference.  Current levels are aligned to known model levels from measurements using ATP.  Error bars are the standard deviation of the mean currents of levels from each molecule measured.  \SI{1}{\Molar} KCl, \SI{2}{\m\Molar} ADPNP, \SI{2}{\m\Molar} MgCl$_2$, \SI{37}{\celsius}.}
\label{fig:helicase_force_stepping_2}
\end{centering}
\end{figure}

The data from several molecules can be aligned to current levels from a known model, as is shown in Figure \ref{fig:helicase_force_stepping_2}.  Shown are the first \num{64} levels from a longer sequence made by ligating Appendix \ref{sequences} \# 0002 and its complement to \# 0003\_ hairpin.  Measured current levels due to forced unzipping (red) align well with model levels (black), again indicating that the helicase is taking single-base steps.

\subsection{Effect of a non-hydrolyzable ATP analog}

Experiments were carried out to ascertain whether or not the presence of a non-hydrolyzable ATP analog made any difference in the stepping behavior of the helicase.  Since helicase has an ATP binding pocket, and undergoes a conformational change upon ATP binding, it is natural to assume that the change in conformation might cause a change in the stepping behavior under applied force.  The ATP analog used in these experiments is adenylyl-imidodiphosphate [ADPNP] (alternatively abbreviated AMP-PNP).  It is similar to ATP in all but the final phosphate, which is linked to the second phosphate via a nitrogen instead of an oxygen, and so it cannot be hydrolyzed.

Figure \ref{fig:helicase_sliding_analog} shows histograms of the times it takes a full DNA molecule to be traversed by Dda helicase using force instead of ATP.  The DNA used here is Appendix \ref{sequences} \# 0002 with complement and its tether, having a dsDNA duplex region of \num{50} base pairs in front of the helicase.  Panel (a) compares the effects of \SI{160}{\mV} applied bias to \SI{180}{\mV} applied bias.  It cam be seen that the mean unzipping time is shorter at the higher bias, but both distributions are quite broad.  A log-normal fit is plotted as a dashed line to guide the eye.

Figure \ref{fig:helicase_sliding_analog}b shows the same histograms when \SI{2}{\m\Molar} ADPNP is added.  The distributions tighten up somewhat compared to no ADPNP; however, they are still quite broad, spanning a range of over \SI{100}{\s} in the time it takes to unzip \num{50} base pairs.  Because the presence of ADPNP seems to result in a tighter distribution of transit times, \SI{2}{\m\Molar} ADPNP is included in the experiments presented in the rest of this work.

\begin{figure}[H]
\begin{centering}
\includegraphics[width=0.8\textwidth]{figures/helicase_sliding_analog.pdf}
\caption[Forced unzipping time depends on the presence of an ATP analog]{(a) Without ATP analog, the distribution of the time it takes to force a helicase to unzip a \num{50}bp duplex has a long tail.  The average of the \SI{180}{\mV} distribution (red) is a shorter time than the \SI{160}{\mV} distribution (blue), but both are quite broad.  (b) With \SI{2}{\m\Molar} ADPNP present, the same distributions become slightly less broad.  Data taken with M3-MspA, \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, \SI{2}{\m\Molar} MgCl$_2$, pH \num{8.00}, \SI{37}{\celsius}.  Log-normal fits are shown as dotted lines to guide the eye.}
\label{fig:helicase_sliding_analog}
\end{centering}
\end{figure}

Due to the single-base resolution obtained from the current versus time traces recorded in these nanopore experiments, it is possible to look at the durations of individual steps of the helicase.  Shown in Figure \ref{fig:helicase_sliding_time_dist} is the probability of a step lasting for a given duration, plotted at three applied bias voltages: \SI{120}{\mV}, \SI{160}{\mV}, and \SI{180}{\mV}.  Note that the data are plotted on a log-log scale.  The differences between the distributions at the different biases are enough to shift the mean lower at higher applied bias.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.5\textwidth]{figures/helicase_sliding_step_dist.pdf}
\caption[Distribution of step durations using mechanical force]{Probability distributions for helicase step durations measured without ATP, when Dda is advanced by mechanical force through a duplex.  The three voltages shown all display quite broad distributions, but step durations are generally shorter at higher applied bias voltages.  M3-MspA, \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, \SI{2}{\m\Molar} MgCl$_2$, pH \num{8.00}, \SI{2}{\m\Molar} ADPNP, \SI{37}{\celsius}.}
\label{fig:helicase_sliding_time_dist}
\end{centering}
\end{figure}

\section{The effects of voltage and temperature}

In an effort to control the motion of the helicase enzyme on DNA, it is important to understand the dependence of the stepping rate on applied voltage and temperature.  The electrophoretic force applied to the ssDNA is directly proportional to the applied voltage, according to $F = \sigma_{\text{eff}} V$.  The quantity $\sigma_{\text{eff}}$ for ssDNA in the MspA nanopore was measured in Chapter \ref{dna_thermal_motion_mspa}, and its average value (over the four nucleotides) can be expressed as \SI{0.177}{\pico\N / \mV} (see Table \ref{table:fit_values}).  This means that the applied force at a bias of \SI{160}{\mV} is approximately \SI{28}{\pico\N}, and that applied biases of \SI{120}{\mV} and \SI{200}{\mV} differ in applied force by \SI{14}{\pico\N}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/helicase_sliding_voltage.pdf}
\caption[Forced helicase stepping depends on voltage]{The median step duration of Dda helicase forced to unwind a duplex is plotted as a function of voltage, at several temperatures.  Spline curves are drawn to guide the eye along measurements at constant temperature.  Error bars are $median(t)/\sqrt{N}$, where $N$ was typically many hundreds of individual steps.  M3-MspA, \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, \SI{2}{\m\Molar} MgCl$_2$, pH \num{8.00}, \SI{2}{\m\Molar} ADPNP.  The voltage biases on the x-axis span the range of approximately \SI{21}{\pico\N} to \SI{35}{\pico\N}.}
\label{fig:helicase_stepping_voltage}
\end{centering}
\end{figure}

Figure \ref{fig:helicase_stepping_voltage} shows the measured step durations as a function of applied constant voltage at various temperatures.  Step durations are characterized by median step duration.  The distribution of the step durations that gives rise to each data point is quite broad, as mentioned in connection with Figure \ref{fig:helicase_sliding_time_dist}.  The error bars correspond only to $median(t)/\sqrt{N}$, and do not reflect the width of the underlying distributions.  In enzymology, this sort of data is typically plotted as a ``force - velocity curve."

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.7\textwidth]{figures/helicase_sliding_temp.pdf}
\caption[Forced helicase stepping depends on temperature]{Same data shown in Figure \ref{fig:helicase_stepping_voltage}.  The median step duration of Dda helicase forced to unwind a duplex is plotted here as a function of temperature, at several applied bias voltages.  Spline curves are drawn to guide the eye along measurements at constant voltage.  Error bars are $median(t)/\sqrt{N}$, where $N$ was typically many hundreds of individual steps.  M3-MspA, \SI{1}{\Molar} KCl, \SI{25}{\m\Molar} K-phosphate, \SI{2}{\m\Molar} MgCl$_2$, pH \num{8.00}, \SI{2}{\m\Molar} ADPNP.}
\label{fig:helicase_stepping_temp}
\end{centering}
\end{figure}

The same data are re-plotted in Figure \ref{fig:helicase_stepping_temp}, this time as an explicit function of temperature.  It is clear from this plot that temperature has a very large impact on the stepping of the helicase, even changes as small as \SI{10}{\celsius} or less.  At some point, the temperature will become too high for the helicase to function, and the protein could even denature.  \SI{50}{\celsius} was the highest temperature studied here, and the helicase still seemed to function as normal.  Higher temperatures were not easily accessible with this experimental setup due to the formation of bubbles which would often destroy the fragile lipid membrane.


\section{Attempts at temporal control using voltage pulses}

\subsection{The physical picture}

The movement of helicase along DNA is a stochastic process, with or without ATP present.  ATP of course facilitates the stepping of helicase, but stepping can still occur without ATP on much longer time scales when force is applied.  One picture consistent with these observations and the crystal structure of Dda and other similar helicases is that the Dda helicase has to wait for a large fluctuation in order to undergo the conformational change required in order to take a step.  The binding of ATP and its subsequent hydrolysis facilitate these conformational changes and increase their rates.  In the absence of ATP, the helicase has some low probability of undergoing a large enough fluctuation to undergo the necessary conformational change.  However, the temperature apparently increases the stepping rate, probably by increasing the size of fluctuations.  The applied force also apparently increases the stepping rate, to a lesser extent, possibly by increasing the frequency with which attempts are made at taking a step.  It is also possible that the helicase must wait for the final base pair of the dsDNA duplex to ``breathe" open, and that this could also limit the stepping rate.

In any event, the probably of helicase stepping increases at increased applied force.  The aim of the work in this section is to increase the applied force for a large amount, but for only a short duration.  The aim is to greatly increase the stepping probability while a short-duration force is applied, and thereby control when the helicase steps.

\subsection{Experimental setup and data}

This brief increase in electrophoretic force is applied using a low-noise voltage pulse generator.  Due to the unacceptable amount of current noise introduced into the experiment by commercial pulse generators that plug into a wall socket, it was necessary to design and build a low-noise pulse generator for this purpose.  Details of the design are included in Appendix \ref{pulse_generator}.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.9\textwidth]{figures/pulse_electronics.pdf}
\caption[Electronic setup for low-noise pulses across a nanopore]{Schematic of the electronic setup used to apply short voltage pulses across the nanopore.  The nanopore equivalent circuit is shown on the bottom left, with access resistance $R_{\text{a}}$, pore resistance $R_{\text{pore}}$, and membrane capacitance $C_{\text{mem}}$.  The voltage difference across the membrane is $\Delta V = V_{\text{bias}} + V_{\text{pulse}}$.  Voltage pulses of duration \SI{50}{\ns} to \SI{10}{\ms} and up to \SI{5}{\V} are applied using a custom-built low-noise pulse generator, inserted between the cis side of the experiment and the ground of the Axopatch's headstage.  In order to cancel out capacitive current spikes before they reach the current amplifier, the pulse generator simultaneously outputs a pulse of opposite polarity, meant to compensate.}
\label{fig:helicase_pulse_setup}
\end{centering}
\end{figure}

Experiments and data

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_pulse_level_change.pdf}
\caption[Voltage pulse can induce a helicase step]{M3-MspA, applied constant bias voltage of \SI{160}{\mV} in \SI{1}{\Molar} KCl, \SI{2}{\m\Molar} ADPNP, at \SI{37}{\celsius}. Data filtered to \SI{200}{\Hz}}
\label{fig:helicase_pulse}
\end{centering}
\end{figure}

\begin{figure}[h]
\begin{centering}
\includegraphics[width=\textwidth]{figures/helicase_data_pulses.pdf}
\caption[A fraction of pulses move the helicase]{M3-MspA, \SI{1}{\Molar} KCl, \SI{2}{\m\Molar} ADPNP, at \SI{37}{\celsius}.  Gray is raw data, hardware filtered at \SI{10}{\kHz}, black is \SI{1}{\kHz} filtered data.  Red vertical lines indicate the timing of pulses, approximately \SI{104}{\ms} apart, \SI{1}{\ms} in duration, and \SI{300}{\mV} in magnitude, in addition to a constant bias voltage of \SI{160}{\mV}.  Green arrows indicate pulses that coincide with changes in current level.}
\label{fig:helicase_pulse_traces}
\end{centering}
\end{figure}

\section{Analysis and discussion}

discussion.

\begin{figure}[h]
\begin{centering}
\includegraphics[width=0.75\textwidth]{figures/helicase_pulse_efficacy.pdf}
\caption[Quantifying efficacy of pulses]{M3-MspA, applied constant bias voltage of \SI{160}{\mV} in \SI{1}{\Molar} KCl, \SI{2}{\m\Molar} ADPNP, at \SI{37}{\celsius}.  Error bars are the standard error of the mean for several individual DNA/helicase complexes.  The black dashed line shows the mean value corresponding to random coincidence of a pulse with a level change, as measured in the \SI{0}{\ms} control experiment.}
\label{fig:helicase_pulse_efficacy}
\end{centering}
\end{figure}

Many pulse voltages and durations were tried, from 200mV to 2V, and 200ns to 2ms, in both 1M KCl and 0.5M KCl, with and without ATP and ADPNP.

Kinesin velocity is independent of force when a constant assisting force is applied.  Velocity of RNA polymerase is nearly force independent \citep{Bustamante2004}.

What really remains is to try other enzymes.
